AWSSDKSwiftCLI/Tests/AWSSDKSwiftCLITests/CLITestCase.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+@testable import AWSSDKSwiftCLI
+import XCTest
+
+class CLITestCase: XCTestCase {
+    let tmpPath = "tmp"
```

Comment:
worth using https://developer.apple.com/documentation/foundation/filemanager/1642996-temporarydirectory ?

that way you don't have to worry about if for some reason cleanup fails. Which I'm sure will when we have more frequent releases (ie more failures too)

Reply:
The current `tmp` directory is cleaned up on setup, so no issues there. And its nbd if a rogue tmp directory is left around, since its located in the application build output directory anyways.